src/component/footer/index.js

```diff
@@ -0,0 +1,2 @@
+export * from './Footer1'
```

Comment:
or you can create a component,

const CustomFooter=()=>{

return(
<Footer1/>
<Footer2/>
)}

export default CustomFooter